spot-vaults/contracts/BillBroker.sol

```diff
@@ -274,7 +279,12 @@ contract BillBroker is
         usd.safeTransferFrom(_msgSender(), address(this), usdAmtIn);
         perp.safeTransferFrom(_msgSender(), address(this), perpAmtIn);
 
-        // mint LP tokens
+        // Permanently lock the MINIMUM_LIQUIDITY tokens on first mint
+        if (isFirstMint) {
+            _mint(address(this), MINIMUM_LIQUIDITY);
+        }
+
+        // mint LP tokens to the user
```

Comment:
I think we need to subtract MINIMUM_LIQUIDITY from the amount minted to user, if it's the first mint.

(edit: answered below)